R/utility.R

```diff
@@ -528,3 +528,7 @@ waitForTasksToComplete <- function(jobId, timeout) {
 
   stop("A timeout has occurred when waiting for tasks to complete.")
 }
+
+areEqual <- function(a, b) {
+  !is.null(a) && !is.null(b) && a == b
```

Comment:
Nitpick: technically null == null.

Also, not sure how R handles this, but do we need to do a shallow or deep comparison on the objects?

Reply:
```if (NULL == 'a') 1```
# Error in if (NULL == "a") 1 : argument is of length zero

Reply:
Just rename the function to ```areShallowEqual```